package.json

```diff
@@ -55,7 +55,7 @@
     "ember-resolver": "^4.0.0",
     "ember-simple-auth": "^1.6.0",
     "ember-sinon": "^2.1.0",
-    "ember-source": "~3.1.0",
+    "ember-source": "~2.16.0",
```

Comment:
@fenichelar why the downgrade here?

Reply:
I didn't do it, I removed `node_modules` and ran `npm install` and it was done automatically. I'll fix.

Reply:
I should have read the version number more carefully... lol

Reply:
Fixed and Travis approves!